galileo/src/lib.rs

```diff
@@ -3,7 +3,63 @@
 //!
 //! # Quick start
 //!
-//! TODO
+//! **Cargo.toml:**
+//!```toml
+//! 
+//! [package]
+//! name = "map-view"
+//! version = "0.1.0"
+//! edition = "2024"
+//! 
+//! [dependencies]
+//! galileo = {git = "https://github.com/maximkaaa/galileo"}
+//! galileo-types = {git = "https://github.com/maximkaaa/galileo"}
+//! galileo-egui = {git = "https://github.com/maximkaaa/galileo"}
+//!```
+//!
+//! **src/main.rs:**
+//! ```rust
+//! use galileo::layer::raster_tile_layer::RasterTileLayerBuilder;
+//! use galileo::{Map, MapBuilder, TileSchema };
+//! use galileo::layer::FeatureLayer;
+//! use galileo::symbol::CirclePointSymbol;
+//! use galileo::galileo_types::latlon;
+//! use galileo_types::geo::Crs;
+//! use galileo::Color;
+//! 
+//! fn main() {
+//!     run()
+//! }
+//! 
+//! // Creates the window and starts the egui event loop.
+//! pub(crate) fn run() {
+//!     galileo_egui::init(create_map(), [])
+//!         .expect("Couldn't create window");
+//! }
+//! 
+//! // build a map showing the Open Street Map raster data for Seoul, South Korea
+//! fn create_map()->Map {
+//!     MapBuilder::default()
+//!         // Set the position of the center of the map view
+//!         .with_position(latlon!(37.566, 126.9784))
+//!         // Set the size of the smallest visible feature (zoom level) for the view to start
+//!         // with.
+//!         .with_resolution(TileSchema::web(18).lod_resolution(8).unwrap())
```

Comment:
There's a new method `with_z_level(8)` now, which is simpler to use in this case.